### PR TITLE
Adds UI wiring for Taxes chart and summary

### PR DESCRIPTION
**Pull request by belcherj**
Wires up the taxes page to display the chart and summary.

Detailed test instructions:
- Checkout branch
- Goto Taxes page
- Not much will appear since swagger API is limited

### CODE REVIEW

**Comment by allendav** [comment]
> TODO: The API appears non-functional at the moment

Looks like the taxes REST API controller is referencing WC_Reports_Orders_Stats_Query when it should be a Taxes Query class (that does not exist.)

File: client/store/reports/stats/resolvers.js [hunk 1]
```diff
@@ -27,6 +27,21 @@ export default {
 
 		let apiPath = endpoint + stringifyQuery( query );
 
+		// TODO: Remove once swagger endpoints are phased out.
```
Comment (timmyc) [inline client/store/reports/stats/resolvers.js]: Nice 👍

File: client/store/taxes/selectors.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { getJsonString } from 'store/utils';
+/**
+ * Returns taxes report details for a specific report query.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} query  Report query paremters
+ * @return {Object}        Report details
+ */
+function getTaxes( state, query = {} ) {
+	return get( state, [ 'taxes', getJsonString( query ) ], [] );
```
Comment (timmyc) [inline client/store/taxes/selectors.js]: Should the default return here be an object `{}` not an array?

File: client/store/taxes/selectors.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+export default {
+	getTaxes,
+	/**
+	 * Returns true if a taxes request is pending.
+	 *
+	 * @param  {Object} state  Current state
+	 * @return {Object}        True if the `getTaxes` request is pending, false otherwise
```
Comment (timmyc) [inline client/store/taxes/selectors.js]: should this be Boolean instead of Object?

File: client/store/taxes/selectors.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	 */
+	isGetTaxesRequesting( state, ...args ) {
+		return select( 'core/data' ).isResolving( 'wc-admin', 'getTaxes', args );
+	},
+	/**
+	 * Returns true if a taxes request has returned an error.
+	 *
+	 * @param  {Object} state  Current state
+	 * @param  {Object} query  Report query paremters
+	 * @return {Object}        True if the `getTaxes` request has failed, false otherwise
```
Comment (timmyc) [inline client/store/taxes/selectors.js]: Likewise Boolean

**Review by timmyc — COMMENTED** [review]
Couple of minor comments but overall this is looking and testing good to me.

File: client/store/taxes/selectors.js (hunk 3 above)
Comment (belcherj) [reply client/store/taxes/selectors.js]: Done

File: client/store/taxes/selectors.js (hunk 2 above)
Comment (belcherj) [reply client/store/taxes/selectors.js]: Done

File: client/store/taxes/selectors.js (hunk 4 above)
Comment (belcherj) [reply client/store/taxes/selectors.js]: Done

**Review by timmyc — APPROVED** [review]
Thanks for the updates. LGTM. Did we determine why CI is failing on this branch though?

**Pull request merged**